src/main/java/Duke.java

```diff
@@ -8,7 +10,18 @@ public static void main(String[] args) {
         System.out.println("Hello from\n" + logo);*/
         System.out.println("Hello! I'm Duke");
         System.out.println("What can I do for you?");
-        System.out.println("Bye. Hope to see you again soon!");
 
+        while (true) {
+            Scanner input = new Scanner(System.in);
+
+            String word = input.nextLine();
```

Comment:
"word" might not be appropriate as it is an input of string that may be a phrase/sentence. Maybe use "userInput"?